server/market/bookrouter.go

```diff
@@ -558,7 +558,7 @@ func (r *BookRouter) msgOrderBook(book *msgBook) *msgjson.OrderBook {
 		MarketID:     book.name,
 		Epoch:        uint64(epochIdx),
 		Orders:       ords,
-		BaseFeeRate:  r.feeSource.LastRate(book.baseID),
+		BaseFeeRate:  r.feeSource.LastRate(book.baseID), // MaxFeeRate applied inside feeSource
 		QuoteFeeRate: r.feeSource.LastRate(book.quoteID),
```

Comment:
I couldn't find a justification for grabbing the `MaxFeeRate` and applying it in the caller (here) instead of in `LastRate`.  This is just easier.

I've documented the `(*feeFetcher).MaxFeeRate` to explain why it still exists despite being applied internally.

Reply:
Definitely cleaner this way.